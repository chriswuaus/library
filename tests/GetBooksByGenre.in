genre Fantasy
add collection test.csv
genre Yeehaw Cowboys
genre Fantasy
add collection duplicates.csv
genre Fantasy
eXIT
