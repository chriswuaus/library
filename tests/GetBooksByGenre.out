user: No books in system.

user: 17 books successfully added.

user: No books with genre Yeehaw Cowboys.

user: The Hobbit (J.R.R. Tolkien)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)

user: 14 books successfully added.

user: The Hobbit (J.R.R. Tolkien)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
The Hobbit (J.R.R. Tolkien)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)

user: Ending Library process.
